scalableminds/webknossos
# Add possibility to edit min and max of intensity range

This PR adds the possibility for the user to edit the minimum and maximum possible value for the intensity range. 
The use case is described in #4526.

### URL of deployed dev instance (used for testing):
- https://addminmaxediting.webknossos.xyz

### Steps to test:
- Open a dataset (view mode is enough)
- Toggle the edit function for min and max. The button is on the left of the reload-layer button.
- enter some desired values
- the histogram and the `intensityRange` slider's range should adjust to the entered values.
- then adjust the `intensityRange` and you should be able to see something again :)

- please try breaking the feature by entering some random and uncommon values

### Issues:
- fixes #4526

------
